src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3540,6 +3540,35 @@
     public static final int COMMENT_CONTENT =
             GeneratedJavaTokenTypes.COMMENT_CONTENT;
 
+    /**
+     * A variable declared in a pattern definition; when conditionally matched,
+     * the variable is assigned to the defined type.
+     *
+     * <pre>
+     * <p>For example:</p>
```

Comment:
```suggestion
     * <p>For example:</p>
     * <pre>
```

Reply:
It looks like this is what is breaking the [Travis build](https://travis-ci.org/github/checkstyle/checkstyle/jobs/704300059#L3179)

Reply:
Fixed.